src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/convert_matmul_to_fc.cpp

```diff
@@ -14,7 +14,8 @@
 ov::intel_cpu::ConvertMatMulToFC::ConvertMatMulToFC() {
     MATCHER_SCOPE(ConvertMatMulToFC);
     auto activations_m = ngraph::pattern::any_input(ngraph::pattern::has_static_rank());
-    auto weights_m = ngraph::pattern::wrap_type<ngraph::opset1::Constant>();
+    auto weights_m = ngraph::pattern::any_input(ngraph::pattern::has_static_rank());
```

Comment:
do we have a test when `weights_m` is not a Constant here?

Reply:
added tests here `convert_matmul_test.cpp`, could you review please?